src/test/librbd/fsx.cc

```diff
@@ -2393,12 +2393,16 @@ test(void)
 			log4(OP_SKIPPED, OP_WRITESAME, offset, size);
 			goto out;
 		}
+		break;
         case OP_COMPARE_AND_WRITE:
                 /* compare_and_write not implemented */
                 if (!ops->compare_and_write) {
                         log4(OP_SKIPPED, OP_COMPARE_AND_WRITE, offset, size);
                         goto out;
                 }
+		break;
+	default:
```

Comment:
@dillaman Do we need to leave a comment about `why there is no default` at L2404, so that some others won't introduce a `default:`, without knowing, in the future ?